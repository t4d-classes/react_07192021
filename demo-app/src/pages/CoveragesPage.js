

export const CoveragesPage = () => (
  <div>
    <header>
      <h1>Coverages</h1>
    </header>
    <p>Manage coverages...</p>
  </div>
);