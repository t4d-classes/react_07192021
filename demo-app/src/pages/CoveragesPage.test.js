import React from "react";

import renderer from "react-test-renderer";

import { CoveragesPage } from "./CoveragesPage";

jest.mock('../hooks/useCoverages');
jest.mock('../components/CoverageList');
jest.mock('../components/CoverageForm');

test("snapshot CoveragesPage component", () => {
  expect(
    renderer.create(
      <CoveragesPage />
    ).toJSON()
  ).toMatchSnapshot();
});
