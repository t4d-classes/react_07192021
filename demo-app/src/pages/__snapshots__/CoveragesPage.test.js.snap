// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot CoveragesPage component 1`] = `
<div>
  <header>
    <h1>
      Coverages
    </h1>
  </header>
  <p>
    Manage coverages...
  </p>
  <form
    className="container-fluid"
  >
    <span>
      Form is Valid
    </span>
    
    <div
      className="row"
    >
      <div
        className="col-md-12 form-group lmgbi_formEntry"
      >
        <label
          className="lmgbi_formTitle"
          htmlFor="coverage-name-input"
        >
          Coverage Name:
        </label>
        <span
          className="required-marker"
        />
        <br />
        <input
          className="form-control lmgbi_formValue lmgbi_formRequired lmgbi_useNameAttr"
          id="coverage-name-input"
          name="coverageName"
          onChange={[Function]}
          type="text"
          value=""
        />
        <br />
        <span
          className="lmgbi_formAlert"
          style={
            Object {
              "color": "red",
              "display": "none",
            }
          }
        >
          Coverage Name required
        </span>
      </div>
    </div>
    <div
      className="row"
    >
      <button
        onClick={[Function]}
        type="button"
      >
        Add Coverage
      </button>
    </div>
  </form>
</div>
`;
